FSI/Displacement.C

```diff
@@ -30,34 +43,45 @@ void preciceAdapter::FSI::Displacement::write(double * buffer, bool meshConnecti
         << exit(FatalError);
 }
 
+
 // return the displacement to use later in the velocity?
 void preciceAdapter::FSI::Displacement::read(double * buffer, const unsigned int dim)
 {
-    // For every element in the buffer
-    int bufferIndex = 0;
-
-    // For every boundary patch of the interface
-    for (uint j = 0; j < patchIDs_.size(); j++)
+    for (unsigned int j = 0; j < patchIDs_.size(); j++)
```

Comment:
we could also use `std::size_t` or `auto` here. But let's be consistent with the rest of the codebase for now.

Reply:
I think the range based solution is more recent, also in OpenFOAM itself. At this specific point, we cannot use it since we need the actual position of the iterator.